controller/src/main.rs

```diff
@@ -322,7 +334,12 @@ impl Application {
 
         for hack in self.enhancements.iter() {
             let hack = hack.borrow();
-            hack.render(&*settings, ui, &self.view_controller);
+            hack.render(RenderContext {
```

Comment:
Construct the `RenderContext` outside of the loop and pass it by reference instead of constructing & moving it every time.

Reply:
This has not been resolved :)